src/main/java/com/litequizapp/server/LiteQuizApplication.java

```diff
@@ -1,50 +1,17 @@
 package com.litequizapp.server;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.boot.CommandLineRunner;
+/*import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;*/
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class LiteQuizApplication {
 
-  private static final Logger log = LoggerFactory.getLogger(LiteQuizApplication.class);
+/*  private static final Logger log = LoggerFactory.getLogger(LiteQuizApplication.class);*/
```

Comment:
In general it's bad idea to keep commented code in your repo.You can find this snapshot in git history if you need it.